doc/content/getting-started/migrating-from-v2/major-changes.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Major Changes in The Things Stack
+weight: 10
+---
+
+Before getting started, we will discuss major breaking changes between
+{{% ttnv2 %}} and {{% tts %}}, along with some guidelines to make the
+migration process easier to manage.
+
+### Application Data
```

Comment:
I don't think we should start with this. I would go from architecture, to gateway, network to application-layer changes.

Reply:
Updated, not sure how much we want to write about the changes that are not entirely related to migrating existing applications and workloads.

Reply:
This is heading `##` and MQTT, payload format, storage etc is al `###` (like now)